### ISSUE TITLE
[Mac research] Run frozen Qwen3 0.6B Benchmark V0 host baseline

### ISSUE THREAD

**Issue by pokekarten** [pokekarten/igentic-iphone#371]
Execution-only host benchmark corresponding to research issue #311. Use the frozen vetted repository SHA and existing benchmark/packager contracts. Run Router-small first and Router-normal only after provenance-clean success. No model adaptation, downloads beyond already authorized runner behavior, prompt/evaluator changes, or product integration.

**Comment by pokekarten** [comment]
{"schema_version":"neo-codex-ready-v1","id":"ig-qwen311-286681ae","repository":"pokekarten/igentic-iphone","issue_number":371,"ref":"286681ae5dbfc1d0455cab248049f166d96c5642","task":"Execute the frozen untouched Qwen3 0.6B Benchmark V0 host baseline from research issue #311 at exact repository SHA 286681ae5dbfc1d0455cab248049f166d96c5642. Read current AGENTS.md and the issue contract. Use the existing pinned runner and deterministic evidence packager only. Run Router-small with the precommitted seed set 0..4; package/validate every completed seed; only if provenance-clean, run Router-normal under the same protocol. Do not change benchmark inputs, prompts, evaluator, model, output repair, integration code, or physical-device claims. If the exact model snapshot/runtime is unavailable, report the precise environment blocker and stop; do not auto-download an unapproved substitute.","timeout_seconds":3600,"execution_profile":"balanced"}

**Comment by pokekartenkiste** [comment]
{"codex_execution_profile":"balanced","codex_fast_mode":false,"codex_invoked":true,"codex_model":"gpt-5.6-terra","codex_reasoning_effort":"medium","codex_usage":{"available":true,"cache_write_input_tokens":0,"cached_input_tokens":150016,"input_tokens":187634,"output_tokens":2123,"reasoning_output_tokens":780,"turns":1},"control_sha":"2bcc355874898a13198939407722ef2cf0c4a9a2","declared_outcome":"BLOCKED","executor":"codex","exit_code":0,"final_message":"WORKER_OUTCOME: BLOCKED\n\nDone: Read the repository contracts and attempted the required offline preflight at SHA `286681ae5dbfc1d0455cab248049f166d96c5642`. No benchmark artifacts were created.\n\nEvidence/tests: Default host Python fails `import torch` (`ModuleNotFoundError`). The exact pinned snapshot `c1899de289a04d12100db370d81485cdf75e47ca` is absent from both standard local Hugging Face cache locations. Worktree remains clean.\n\nStill open: Router-small seeds 0\u20134, their evidence packages, provenance validation, and Router-normal are unexecuted.\n\nNext causal action: Provision an already-downloaded exact Qwen snapshot and a local PyTorch + Transformers \u22654.51 runtime; then rerun the pinned offline runner.","finished_at":"2026-09-06T12:03:03.922545+00:00","handoff_id":"ig-qwen311-286681ae","host_preflight":{"checked_at":"2026-09-06T12:01:56.452519+00:00","coordination":[],"live_main_sha":"d963479e8e4bf27332bf367a6cee3e445dfe278a","open_pull_requests":[{"baseRefName":"main","headRefName":"dependabot/github_actions/actions/upload-artifact-7","number":370,"url":"https://github.com/pokekarten/igentic-iphone/pull/370"},{"baseRefName":"main","headRefName":"dependabot/github_actions/actions/checkout-7","number":369,"url":"https://github.com/pokekarten/igentic-iphone/pull/369"}],"repository":"pokekarten/igentic-iphone","schema_version":"neo-host-preflight-v1","source_issue_number":371,"task_start_commit":"286681ae5dbfc1d0455cab248049f166d96c5642"},"repository":"pokekarten/igentic-iphone","requested_ref":"286681ae5dbfc1d0455cab248049f166d96c5642","route":"codex","schema_version":"neo-mac-result-v1","source":"github_issue","source_issue_number":371,"state":"BLOCKED","stderr_content_recorded":false,"stderr_sha256":"e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855","task_branch":"codex/ig-qwen311-286681ae","task_commit_sha":null,"task_delivery_error":null,"task_delivery_state":"NO_CHANGES","task_pr_url":null,"task_remote_branch_pushed":false,"task_start_commit":"286681ae5dbfc1d0455cab248049f166d96c5642","task_workspace_cleanup_error":null,"task_workspace_retained":false,"worker_invocation_source":"launchagent","worktree_clean_after":true}

**Comment by pokekarten** [comment]
Closing this execution wrapper on its allowed terminal blocker. The exact frozen checkout was used, but the required local PyTorch/Transformers runtime and exact pinned Qwen snapshot were unavailable; no benchmark artifacts were created and no substitute model/runtime was used. Parent research issue #311 now records the same terminal fail-closed outcome and is closed accordingly. A future benchmark attempt should be a fresh authorized host campaign rather than reopening this stale wrapper.
